bigquery/simpleapp/simpleapp.go

```diff
@@ -42,7 +42,7 @@ func main() {
 
 	client, err := bigquery.NewClient(ctx, projectID)
 	if err != nil {
-		log.Fatalf("bigquery.NewClient: %v", err)
+		log.Fatalf("bigquery.NewClient: %w", err)
```

Comment:
The `%w` is not supposed to do anything special in `Fatalf` (`Printf`), so no need to change these usages.

Reply:
"log.Fatalf does not support error-wrapping directive %w" - linter in VS Code

Reply:
oops, good catch. That was a mistake that was undone locally and when I came back to the computer I saw that the file was not saved 🤦